pages/workspace/design-panel-page.js

```diff
@@ -1090,9 +1090,8 @@ exports.DesignPanelPage = class DesignPanelPage extends BasePage {
   }
 
   async isAnnotationAddedToComponent(value) {
-    await this.page.waitForTimeout(500);
+    await this.annotationCreateTitle.waitFor({ state : 'visible' });
```

Comment:
I'd change it in favor of the locator toBeVisible assertion, as in the two previous comments.

Reply:
done